distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -2,6 +2,8 @@ from Standard.Base import all
 from Standard.Base.Error.Problem_Behavior import Report_Warning
 
 from Standard.Table import Auto_Detect, Aggregate_Column, Data_Formatter, Column_Name_Mapping, Column_Selector, Sort_Column_Selector, Sort_Column, Match_Columns, Position
+import Standard.Table.Data.Expression.Expression as Derived_Expression
```

Comment:
I feel like this rename can be a source of confusion as we will be in one place referring by name `Expression` and elsewhere `Derived_Expression`. I assume the reason for this rename is the clash with the SQL IR Expression. In that case maybe it will be easier to rename this one (which appears only here) to sth else - like `SQL_Expression` or sth along these lines? Then we will be able to use the same name everywhere for the `Derived_Expression`.

Alternatively, maybe let's just rename `Expression` to `Derived_Expression` everywhere? I just think that it will be much better to be consistent.